buildkite/shared-build-env.dockerfile

```diff
@@ -1,4 +1,9 @@
-FROM vimc/node-docker-openjdk:master
+FROM vimc/node-docker:master
+
+RUN wget -qO - https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public | sudo apt-key add -
+RUN add-apt-repository --yes https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/
+RUN apt-get update
+RUN apt-get install adoptopenjdk-11-hotspot -y
```

Comment:
Vimc docker hub image runs jdk8 and node-docker image, which is incompatible with this upgrade. I have reused the node-docker image and installed adopted jdk11 as seen above. 

I didn't install jdk11 from Debian via `stretch-backports` as we had it before. Upgrading to jdk11 proofed difficult because ssl certificate was removed from jdk11 provided by`deb http://deb.debian.org/debian stretch-backports main `, installing and updating certificates didn't work either so I changed to adoptedjdk11.

Reply:
Looks tedious, I prefer using  `vimc/node-docker:master` baseImage. The image is still working.

Reply:
Either way, should probably update our base image with both java and docker so both can be less tedious. I'll make a ticket for it.

Reply:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-3488